README.md

```diff
@@ -86,7 +86,12 @@ By default:
 * It will be installed inside the `equella-install` folder.
 * The options for autotesting and coverage will be configured already.
 * It will be configured for the Postgres DB `equellatests` at `localhost:5432`, expected a username / password of `equellatests` / `password`.  These details can be changed as desired.
-
+```bash
+sudo -u postgres psql
+ CREATE DATABASE equellatests;
+ CREATE USER equellatests WITH PASSWORD 'password';
+ GRANT ALL PRIVILEGES ON equellatests TO equellatests; 
+```
```

Comment:
This new block seems to be sitting here as a fragment. Maybe you need a bit of a prelude to say something like:

> To manually create this database (assuming debian - or derivative):